### ISSUE TITLE
JS error while rendering graph

### ISSUE THREAD

**Issue by mahendrakariya** [kiali/kiali#2625]
**Describe the bug**

On opening service graph for a particular namespace, the following error is shown.

![image](https://user-images.githubusercontent.com/2538874/78665724-dc804000-78f3-11ea-9221-2cca0efe6e49.png)

- This is happening only for one particular namespace. Graph loads fine for all other namespaces.
- This error can be seen for all variations of service graph.
- There is no major difference between the namespaces, to the best of our knowledge.
- This bug is only in Google Chrome. It works fine on Firefox.

Javascript logs from the browser console.

```
u @ namespaces?namespaces=<ns>&graphType=service&edges=responseTime&unusedNodes=true&injectServiceNodes=true&pi=0&duration=60&layout=cose-bilkent:1
1806 @ main.cdd7e02c.chunk.js:1
u @ namespaces?namespaces=<ns>&graphType=service&edges=responseTime&unusedNodes=true&injectServiceNodes=true&pi=0&duration=60&layout=cose-bilkent:1
f @ namespaces?namespaces=<ns>&graphType=service&edges=responseTime&unusedNodes=true&injectServiceNodes=true&pi=0&duration=60&layout=cose-bilkent:1
e @ namespaces?namespaces=<ns>&graphType=service&edges=responseTime&unusedNodes=true&injectServiceNodes=true&pi=0&duration=60&layout=cose-bilkent:1
(anonymous) @ main.cdd7e02c.chunk.js:1
Show 33 more frames
2.c0e22535.chunk.js:1 TypeError: l.toFixed is not a function
    at t (main.cdd7e02c.chunk.js:1)
    at label (main.cdd7e02c.chunk.js:1)
    at e.Oi.applyParsedProperty (2.c0e22535.chunk.js:1)
    at e.Oi.applyContextStyle (2.c0e22535.chunk.js:1)
    at e.Oi.apply (2.c0e22535.chunk.js:1)
    at Qo.updateStyle (2.c0e22535.chunk.js:1)
    at Li.endBatch (2.c0e22535.chunk.js:1)
    at t.value (main.cdd7e02c.chunk.js:1)
    at t.value (main.cdd7e02c.chunk.js:1)
    at 2.c0e22535.chunk.js:1
```

**Versions used**
Kiali: v1.9.1 (2431f9b8844202e27ec1c92f404fe7485253ffec)
Istio: 1.4.5
Kubernetes flavour and version: v1.14.9-eks
Chrome: 80.0.3987.162 (Official Build) (64-bit) on Ubuntu 18.04

**Expected behavior**
Graph should be rendered without any issue.

**Comment by lucasponce** [comment]
Kiali 1.9.1 sounds old and this error sounds familiar.
Perhaps @jshaughn can comment if this is resolved in an upper version.

**Comment by mahendrakariya** [comment]
Actually this looks like a caching issue. It is working fine if I open it on Chrome's incognito mode.

**Comment by jshaughn** [comment]
This has been seen occasionally and should be resolved in Kiali 1.16.0.  We are still looking for the root cause but we have added protective code that should prevent it from affecting users.
